blog/test.py

```diff
@@ -101,7 +101,21 @@ def test_edit_article(self):
                                   data=data)
         article = Article.objects.first()
         self.assertEqual(article.content, '<p>another content</p>')
-        self.assertEqual(result.status_code, 302)
+        self.assertEqual(result.status_code, 204)
```

Comment:
I'm asking if the endpoint `/article/<string:id>/edit` must be a redirection or not.